dts/bindings/pinctrl/microchip,xec-pinctrl.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2020 Linaro Limited
+# Copyright (c) 2021 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Microchip XEC Pin controller Node
+    Based on pincfg-node.yaml binding.
+
+    Note: `bias-disable` and `drive-push-pull` are default pin configurations.
+           They will be applied in case no `bias-foo` or `driver-bar` properties
+           are set.
+
+compatible: "microchip,xec-pinctrl"
+
+include:
+    - name: base.yaml
+    - name: pincfg-node.yaml
+      child-binding:
+        property-allowlist:
+          - bias-disable
+          - bias-pull-down
+          - bias-pull-up
+          - drive-push-pull
+          - drive-open-drain
+          - output-high
+          - output-low
+
+properties:
+    reg:
+      required: true
+
+child-binding:
+    description: |
+        This binding gives a base representation of the Microchip XEC pins
+        configration
+
+    properties:
+        pinmux:
+            type: int
+            required: true
+            description: Pinmux selection
+
+        slew-rate:
+          required: false
+          type: string
+          default: "hw-default"
+          enum:
+          - "hw-default"        # Default value
+          - "low-speed"
+          - "high-speed"
+          description: Pin speed.
+
+        drive-strength:
+          required: false
+          type: string
+          default: "hw-default"
+          enum:
+            - "hw-default"      # Default value
```

Comment:
ditto

Reply:
I see 2 valid sets, can you please clarify:

![image](https://user-images.githubusercontent.com/25011557/152316332-15d1fb6e-152e-41c7-b7a9-6536af4636a6.png)

Reply:
Table 2-2 indicates the PIO (buffer) type for the pins. The first set of values is for pins with buffer type PIO, the second set of values applies to pins with buffer type PIO-24.

Will update doc to include this information.